### DISCUSSION TITLE
如何去掉隔离对话中的锁

### DISCUSSION THREAD

**Discussion by HahaCodeLife** [nianhua99/PandoraHelper#126] [General]
也就是说对于共享中的任意用户只显示他的自己的聊天记录

**Comment by nianhua99** [comment]
neo 研究过， 直接去掉会导致滚动加载出问题，所以现在的🔒已经是折中方案了

**Reply by HahaCodeLife** [reply]
哦 好的 那现在假如我想要修改🔒为空文本或者空白字符该怎么修改(不移除记录，只是美观上优化)，能不能提供一个接头可以定制

**Reply by nianhua99** [reply]
> 哦 好的 那现在假如我想要修改🔒为空文本或者空白字符该怎么修改(不移除记录，只是美观上优化)，能不能提供一个接头可以定制

可以试试nginx的sub_filter

**Reply by HahaCodeLife** [reply]
> > 哦 好的 那现在假如我想要修改🔒为空文本或者空白字符该怎么修改(不移除记录，只是美观上优化)，能不能提供一个接头可以定制
> 
> 
> 
> 可以试试nginx的sub_filter

好的
